operator/pkg/reconciliation/constructor.go

```diff
@@ -379,6 +379,7 @@ func buildContainers(dc *api.CassandraDatacenter, serverVolumeMounts []corev1.Vo
 		{Name: "MGMT_API_EXPLICIT_START", Value: "true"},
 		// TODO remove this post 1.0
 		{Name: "DSE_MGMT_EXPLICIT_START", Value: "true"},
+		{Name: "IGNORE_DEFAULTS", Value: "true"},
```

Comment:
this needs to go in #129 right?

Reply:
yeah, I just added it in #129.

Reply:
Weren't we going to update the management api to default the value of this to that of `USE_MGMT_API` to retain compatibility?

Reply:
with 0.1.5, this is no longer needed

Reply:
I removed the env var